integration_test/dapp_tests/steak/SteakTests.js

```diff
@@ -0,0 +1,175 @@
+const {
+  storeWasm,
+  deployErc20PointerForCw20,
+  ABI,
+  getEvmAddress,
+  fundSeiAddress,
+  associateKey,
+} = require("../../../contracts/test/lib.js");
+const {
+  getValidators,
+  instantiateHubContract,
+  bond,
+  addAccount,
+  queryTokenBalance,
+  unbond,
+  transferTokens,
+} = require("./utils.js");
+
+const { expect } = require("chai");
+const { v4: uuidv4 } = require("uuid");
+
+const STEAK_HUB_WASM = "./steak/contracts/steak_hub.wasm";
```

Comment:
Might need to be `../integration_tests/dapp_tests/steak/contracts/<>` for both here